fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -809,12 +809,14 @@ private void handleAnalyzeStmt() {
                             analyzeStmt.isSample() ? StatsConstants.AnalyzeType.SAMPLE : StatsConstants.AnalyzeType.FULL,
                             StatsConstants.ScheduleType.ONCE, analyzeStmt.getProperties()));
         }
-
-        if (analyzeStatus.getStatus().equals(StatsConstants.ScheduleStatus.FINISH)) {
-            context.getState().setOk();
-        } else {
-            context.getState().setError(analyzeStatus.getReason());
+        ShowResultSet resultSet = analyzeStatus.toShowResult();
+        if (isProxy) {
+            proxyResultSet = resultSet;
+            context.getState().setEof();
+            return;
         }
```

Comment:
What's this?

Reply:
This is the output information of the executed statement. If it is a follower, it needs to be forwarded.